import/neo4j

```diff
@@ -27,6 +27,7 @@ import os
 import warnings
 
 DEFAULT_ENDPOINT =  'http://neo4j:insecure@localhost:7474/db/data'
+wipeDatabase = True
```

Comment:
hmm.. I'm not sure I want this by default when importing the data in neo4j.. was this intended?